### ISSUE TITLE
[Feature] .removeMessages(int count) 

### ISSUE THREAD

**Issue by ABravePanda** [DV8FromTheWorld/JDA#688]
[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
[stack overflow]: https://stackoverflow.com/questions/tagged/java

## General Troubleshooting

Hey there! Before you report a bug or suggest a new feature,
  please make sure to follow these steps first!
  
- [x] I have checked for similar issues.
- [x] I have updated to the [latest JDA version][download].
- [x] I have checked the branches or the maintainers' PRs for upcoming features/bug fixes.

> This is not the place to learn Java. Please refer to [StackOverflow][stack overflow]
  for your general programming questions.

## Issue
Possible .removeMessages(int count) function

### Issue Type

- [ ] Bug Report
- [x] Feature Request

> For questions join the official JDA discord server: https://discord.gg/0hMr4ce0tIl3SLv5

### Description

So, i'm not exactly sure if this is possible, I haven't found a way to do it. If it is possible.. im sorry im a noob... but anyways,

basically on any "TextChannel" object, theres a void method called .removeMessages(int count) that gets the amount of messages specified in the channel, and deletes them. This would be good for something like a purge command. If this already exists, this could be a way to make it easier..

Anyways, just a suggestion.
Thanks for reading!
!
